pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiOffloadTest.java

```diff
@@ -180,43 +183,32 @@ public void testOffloadPolicies() throws Exception {
     public void testOffloadPoliciesApi() throws Exception {
         final String topicName = testTopic + UUID.randomUUID().toString();
         admin.topics().createPartitionedTopic(topicName, 3);
-        //wait for server init
-        Thread.sleep(1000);
+        pulsarClient.newProducer().topic(topicName).create().close();
+        Awaitility.await().atMost(10, TimeUnit.SECONDS)
```

Comment:
please consider leaving the default wait time (here and below), that is 10 seconds

as @lhotari suggested in #9258

Reply:
Okay, let me extend the timeout period, but based on experience, the initialization will be successful within 2 seconds.